### DISCUSSION TITLE
Too many release assets ?? 🤔📦

### DISCUSSION THREAD

**Discussion by RoiArthurB** [gama-platform/gama#81] [Polls]
@gama-platform/core We start to have a significant number of release because of the number of installers, the number of architecture (wink wink M1) and the JDK embedded or not. 

This all gives us this ~10 files per release : 
- 4 releases for macOS
  - 2 per architecture (both under `.dmg`, we already removed zip)
- 2 releases for Windows
  - May evolve with future `.msi` installer
- 3 releases for Linux
  - 2 `.zip` and 2 `.deb` (but will soon grow to have more distro support)

I think this amount of release can start to be confusing for final users. 
I also remember that we did want to only release version with JDK (which can be a problem in few specific cases).

**Comment by hqnghi88** [comment]
Assets are assets, it should have many as possible. The thing we should do is to transparent them from normal/new user with the universal installer 
<img width="480" alt="Screen Shot 2022-04-19 at 15 58 30" src="https://user-images.githubusercontent.com/6105384/163968696-35b32437-d832-4636-b138-c265014229b4.png">

**Reply by RoiArthurB** [reply]
Unfortunately, it's impossible to propose a file based on the computer's OS from Github Release page :/

**Reply by hqnghi88** [reply]
but we do have official website?

**Reply by RoiArthurB** [reply]
The closest I saw is what the [Cider project](https://github.com/ciderapp/Cider/releases) do with a link release post with a different repo holding each release (with a wild number if you click the `Other platform >>" option

![image](https://user-images.githubusercontent.com/16764085/163969436-67591f7b-6e7f-4daa-afca-a420364ccdde.png)

**Reply by hqnghi88** [reply]
Yes, but maybe forget about pure technical releases. It should be publish on official website as a universal INSTALLER

**Reply by RoiArthurB** [reply]
> but we do have official website?

I'm not sure what we can do (technical tweaks are a bit limited since we're on GH Pages), but yes, it's filtered if you pass by the website.

Nevertheless, it keeps the question about if we still release w/ and w/o JDK versions 🤔🤔🤔
